src/pages/apps/new/[provider]/Provider.spec.js

```diff
@@ -35,19 +35,13 @@ describe.skip("pages/apps/new/[provider]", () => {
     const provider = "github";
 
     beforeEach(() => {
-      global.GITHUB_ACCESS_TOKEN = "access-token";
```

Comment:
**Note** The `global` object is not polyfilled, and since this was not used anymore I simply removed it.